src/components/side-panel/layer-panel/custom-picker.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {SketchPicker} from 'react-color';
+import onClickOutside from 'react-onclickoutside';
+
+// This was put in because 3rd party library react-color doesn't yet cater for customized color of child component <SketchField> which contains HEX/RGB input text box
+// Issue raised: https://github.com/casesandberg/react-color/issues/631
+
+const StyledPicker = styled.div`
+  .sketch-picker {
+    span {
+      color: #6a7485 !important;
+      font-family: ff-clan-web-pro, 'Helvetica Neue', Helvetica, sans-serif !important;
+    }
+    input {
+      text-align: center;
+      font-family: ff-clan-web-pro, 'Helvetica Neue', Helvetica, sans-serif !important;
+      background-color: #3a414c !important;
+      color: #a0a7b4 !important;
+      border-color: #3a414c !important;
+      box-shadow: none !important;
+    }
+  }
+`;
+
+class CustomPicker extends Component {
+  static propTypes = {
+    color: PropTypes.string,
+    onChange: PropTypes.func,
+    onSwatchClose: PropTypes.func
+  };
+
+  handleClickOutside = e => {
+    this.props.onSwatchClose();
+  };
+
+  render() {
+    const {color, onChange} = this.props;
+
+    return (
+      <StyledPicker>
+        <SketchPicker
+          color={color}
+          onChange={onChange}
+          disableAlpha={true}
+          presetColors={[]}
+          styles={{
```

Comment:
store the __styles__ object as constant:
```
const PICKER_STYLES = {
            picker: {
              width: '200px',
              padding: '10px 10px 8px',
              boxSizing: 'initial',
              background: '#29323C'
           },
            controls: {
              display: 'flex',
              color: '#FFF'
            }
}
```
and it will become:
```
styles: {PICKER_STYLES}
```
This will make easier to review styles, change it and improve performance since it will not trigger a re-render

Reply:
ok.